app/main.py

```diff
@@ -1,3 +1,31 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from os import path
```

Comment:
The import statement should be `import datetime` as per the checklist requirement.